README.md

```diff
@@ -5,15 +5,15 @@
 
 # Odyssey Design System
 
-This repository contains the code for Okta's Design System Odyssey. It includes Icons, SCSS, and React Components used to build products and experiences at Okta.
+This repository contains the code for Okta's Design System Odyssey. It includes Icons and React Components used to build products and experiences at Okta.
 
 ## Packages
 
-| Package/README                                                                                      | Description                                                        |
-| --------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ |
-| [@okta/odyssey-icons](https://github.com/okta/odyssey/tree/master/packages/odyssey-icons/README.md) | Odyssey SVG icons.                                                 |
-| [@okta/odyssey-react](https://github.com/okta/odyssey/tree/master/packages/odyssey-react/README.md) | Odyssey components for [React](https://reactjs.org/).              |
-| [@okta/odyssey-scss](https://github.com/okta/odyssey/tree/master/packages/odyssey-scss/README.md)   | Odyssey SCSS, includes components, includes base and reset styles. |
+| Package/README                                                                                                      | Description                                                   |
+| ------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------- |
+| [@okta/odyssey-design-tokens](https://github.com/okta/odyssey/tree/master/packages/odyssey-design-tokens/README.md) | Low level tokens used to describe component style properties. |
```

Comment:
no component styles are described in this package today 😅

Reply:
I didnt mean for that to be how that was interpreted. I can figure out different wording. Maybe:

> Low level token values needed to construct and maintain Odyssey